projects/bp-strapi/package.json

```diff
@@ -34,6 +34,8 @@
     "pg": "8.6.0",
     "pm2": "^5.2.0",
     "sendmail": "^1.6.1",
+    "sharp": "^0.32.1",
```

Comment:
I looked this up and it seems to be related to image optimization. Do we do some sort of image search or why do we need this package?

Reply:
i merged staging into my local branch and and suddenly got errors because the sharp package was missing, so installed it. I don't know where this dependency comes from

Reply:
Strange, for me it works without the package